src/lib/blit.c

```diff
@@ -855,7 +855,7 @@ static struct blitset notcurses_blitters[] = {
      .blit = tria_blit,      .name = "eightstep",     .fill = false, },
    { .geom = NCBLIT_1x1,     .width = 1, .height = 1, .egcs = L" █",
      .blit = tria_blit_ascii,.name = "ascii",         .fill = false, },
-   { .geom = NCBLIT_2x1,     .width = 1, .height = 2, .egcs = L" ▄█",
+   { .geom = NCBLIT_2x1,     .width = 1, .height = 2, .egcs = L" ▄█▀",
      .blit = tria_blit,      .name = "half",          .fill = false, },
    { .geom = NCBLIT_2x2,     .width = 2, .height = 2, .egcs = L" ▗▐▖▄▟▌▙█",
```

Comment:
looks like we still need to expand this egcarray. probably ought get a unit test on this.

Reply:
got quadrants, still need handle sextants, this *really* does need a simple unit test each

Reply:
this is going to break plots for these blitters, which also use `egcs`, but with a different ordering. =[

Reply:
for this last, doubling up the `wchar_t` sets within `blitset`, moving `egcs` to `plotegcs`

Reply:
i'm building up these unit tests now

Reply:
all egcarrays are built out now, and we have unit tests